src/sparsezoo/utils/registry.py

```diff
@@ -149,24 +181,37 @@ def registered_names(cls) -> List[str]:
         """
         return registered_names(cls)
 
+    @classmethod
+    def registered_aliases(cls) -> List[str]:
+        """
+        :return: list of all aliases registered to this class
+        """
+        return registered_aliases(cls)
+
 
 def register(
     parent_class: Type,
     value: Any,
     name: Optional[str] = None,
+    alias: Union[List[str], str, None] = None,
     require_subclass: bool = False,
 ):
     """
     :param parent_class: class to register the name under
     :param value: the value to register
     :param name: name to register the wrapped value as, defaults to value.__name__
+    :param alias: alias or list of aliases to register the wrapped value as,
+        defaults to None
     :param require_subclass: require that value is a subclass of the class this
         method is called from
     """
     if name is None:
         # default name
         name = value.__name__
 
+    name = standardize_lookup_name(name)
+    register_alias(name=name, alias=alias, parent_class=parent_class)
```

Comment:
we need to standardize the aliases as well right?

Reply:
Do we? I thought that I'd be nice (hopefully) not confusing aliases to be more "liberal", i.e. the user can set them directly to their face value, but I do not have a strong opinion there.

Reply:
I'd vote to be consistent and standardize everything, if we standardize the inputs as well then the user wouldn't notice a change right?